src/pretix/control/forms/organizer.py

```diff
@@ -651,7 +651,7 @@ class CustomerCreateForm(CustomerUpdateForm):
 
     class Meta:
         model = Customer
-        fields = ['identifier', 'is_active', 'name_parts', 'email', 'is_verified', 'locale']
+        fields = ['identifier', 'is_active', 'name_parts', 'email', 'is_verified', 'locale', 'external_identifier', 'notes']
```

Comment:
The Phone-Number text-input in the create-view is appended as the last input. Have not found a way yet, to change this – but have not looked that hard for it. Maybe just leave it as is?

Reply:
Hmm, I think honestly that the external identifier should be *above* the name and the notes should be the last one. 

maybe

- identifier
- external_identifier
- is_active
- name_parts
- email
- is_verified
- phone
- locale
- notes

If it's too hard to resort the fields dictionary, the next-best option would be to have the fields explicitly one by one in the template. That's how we do it for every complex form anyways

Reply:
Yes, makes more sense. I changed the order of fields like you suggested, except I moved `is_active` as the first item – it just looked better with the checkbox at the top and felt more „in line“ with other forms we have, where one can activate something.